Makefile

```diff
@@ -135,6 +145,9 @@ uninstall-inventory-scripts:
 	done
 	-rmdir -p $(prefix)$(datadir)/mender/inventory
 
+uninstall-inventory-network-scripts:
+	rm -f $(prefix)$(datadir)/mender/inventory/$(INVENTORY_NETWORK_SCRIPTS)
+
```

Comment:
Also, not very important, but I am missing an equivalent `uninstall-inventory-local-scripts` (sic).